hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/metrics/DataNodeMetrics.java

```diff
@@ -323,10 +323,10 @@ public void addIncrementalBlockReport(long latency,
     }
   }
 
-  public void addReadTransferRate(long readTransferRate) {
-    this.readTransferRate.add(readTransferRate);
-    for (MutableQuantiles q : readTransferRateQuantiles) {
-      q.add(readTransferRate);
+  public void addReadLatencyPerGB(long readLatencyPerGB) {
+    this.readLatencyPerGB.add(readLatencyPerGB);
+    for (MutableQuantiles q : readLatencyPerGBQuantiles) {
```

Comment:
Why don't we modify these quantiles rather than the metric? 

I think we have the metric we want, but we want the `lowest` vs `highest` percentile (ts clear the current percentile is lower=better).